services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_task_utils.py

```diff
@@ -39,7 +39,7 @@ async def apply_observation_cycle(
         )
         await scheduler.mark_service_for_removal(
             node_uuid=scheduler_data.node_uuid,
-            can_save=scheduler_data.dynamic_sidecar.service_removal_state.can_save,
+            can_save=scheduler_data.dynamic_sidecar.were_containers_created,
```

Comment:
so this is the the fix in here?

Reply:
Yes, exactly there. This is a long time bug.